src/material/expansion/expansion-panel.ts

```diff
@@ -185,7 +185,7 @@ export class MatExpansionPanel extends CdkAccordionItem implements AfterContentI
       // We don't need to subscribe to the `stateChanges` of the parent accordion because each time
       // the [displayMode] input changes, the change detection will also cover the host bindings
       // of this expansion panel.
-      return (this.expanded ? this.accordion.displayMode : this._getExpandedState()) === 'default';
+      return this.expanded && this.accordion.displayMode === 'default';
```

Comment:
I'm not sure the comment above it makes sense anymore either.

Reply:
Yeah, I'll remove it since it doesn't seem helpful in that place. I got confused too.